app/assets/stylesheets/sufia/_fixedsticky.scss

```diff
@@ -0,0 +1,28 @@
+.fixedsticky {
```

Comment:
Same question. This seems like a mixed of upstream styles and our own? Should we split the upstream stuff into vendor?